src/plugins/initiatives/declaration.js

```diff
@@ -0,0 +1,112 @@
+// @flow
+
+import deepFreeze from "deep-freeze";
+import type {PluginDeclaration} from "../../analysis/pluginDeclaration";
+import type {NodeType, EdgeType} from "../../analysis/types";
+import {NodeAddress, EdgeAddress} from "../../core/graph";
+
+export const nodePrefix = NodeAddress.fromParts(["sourcecred", "initiatives"]);
+export const edgePrefix = EdgeAddress.fromParts(["sourcecred", "initiatives"]);
+
+export const initiativeNodeType: NodeType = deepFreeze({
+  name: "Initiative",
+  pluralName: "Initiatives",
+  prefix: NodeAddress.append(nodePrefix, "initiative"),
+  defaultWeight: 1,
+  description:
+    "An initiative, formatted as a Discourse wiki topic using a fixed template.",
+});
+
+/*
+	Note on the forward and backward naming convention.
+	It follows the core/graph.js documentation to use
+	a <subject> <verb> <object> format to figure out
+	the directionality.
+*/
+
+/**
+ * A Discourse topic (src) TRACKS (verb) an initiative (dst).
+ * Forward: separate in terms of cred flow for now
+ * Backward: separate in terms of cred flow for now
+ */
+export const discourseTopicTracksInitiativeEdgeType: EdgeType = deepFreeze({
+  forwardName: "tracks",
+  backwardName: "is tracked by",
+  prefix: EdgeAddress.append(edgePrefix, "discourseTopicTracksInitiative"),
+  defaultWeight: {forwards: 0, backwards: 0},
+  description: "Connects an initiative to the Discourse topic that tracks it.",
+});
+
+/**
+ * An initiative (src) DEPENDS ON (verb) a dependency (dst).
+ * Forward: depending on something flows significant cred to the dependency.
+ * Backward: having a dependency does not flow much cred to the iniative,
+ * though does flow some to incentivize reuse and attribution.
+ */
+export const dependsOnEdgeType: EdgeType = deepFreeze({
+  forwardName: "depends on",
+  backwardName: "is a dependency for",
+  prefix: EdgeAddress.append(edgePrefix, "dependsOn"),
+  defaultWeight: {forwards: 1, backwards: 1 / 16},
```

Comment:
I notice we use 1 as "significant" and 1/2 as "medium".

Wondering if we should re-anchor on 1 as medium, and 2 or 4 as "significant".
(This might imply changes to other plugins, so its fine if we don't address it now.)

Reply:
I'm happy to adjust the comments in any way that is more clear. I used them as rational how I thought about value represented by edges. So long as the numerical weights are solid :D

Reply:
I've removed these quantifiers, and instead explained in more relative terms why some edges are assumed to be more important than others.